xtask/src/commands/build.rs

```diff
@@ -19,6 +19,8 @@ use crate::{
 pub enum Build {
     /// Build documentation for the specified chip.
     Documentation(BuildDocumentationArgs),
+    /// Build documentation index.
+    DocumentationIndex,
```

Comment:
Let's feature gate this so it doesn't show up to local users just trying to build the docs (for confusion reasons)